### PR TITLE
Autodict forkserver

### LINKED ISSUE #504: Forkserver AutoTokens
Now we have AutoTokens (autodict) feature for inprocess fuzzer, we also want it for the forkserver executor

### PR DESCRIPTION
**Pull request by tokatoka**
Closes #504 
depends on #523 

this pr adds AUTODICT feature for the forkserver executor.

### CODE REVIEW

**Comment by domenukk** [comment]
@tokatoka this needs merging

File: fuzzers/forkserver_simple/src/main.rs [hunk 1]
```diff
@@ -145,13 +145,11 @@ pub fn main() {
...
     };
 
-    let mut tokens = Tokens::new();
-    let forkserver = ForkserverExecutorBuilder::new()
+    let forkserver = ForkserverExecutor::builder()
         .program(res.value_of("executable").unwrap().to_string())
         .args(&args)
         .debug_child(debug_child)
         .shmem_provider(&mut shmem_provider)
-        .autodict_tokens(&mut tokens)
```
Comment (domenukk) [inline fuzzers/forkserver_simple/src/main.rs]: no more tokens?

Comment (domenukk) [reply fuzzers/forkserver_simple/src/main.rs]: btw why not `auto_tokens`?

File: libafl/src/executors/forkserver.rs [hunk 2]
```diff
@@ -761,10 +774,11 @@ impl<'a> ForkserverExecutorBuilder<'a, StdShMemProvider> {
     }
 
     #[must_use]
-    /// Place the input at this position, this is equivalent to putting "@@" here.
-    pub fn arg_input_file(mut self) -> Self {
```
Comment (domenukk) [inline libafl/src/executors/forkserver.rs]: I'd add an `arg_input_file_std` with default filename (sorry that should have gone in the other PR)

File: fuzzers/forkserver_simple/src/main.rs (hunk 1 above)
Comment (tokatoka) [reply fuzzers/forkserver_simple/src/main.rs]: the instrumentation side is 100% afl++, so I decided to use the original name

Comment (domenukk) [reply fuzzers/forkserver_simple/src/main.rs]: Let's keep it consistent in the project(?)

**Pull request merged**